cloud/endagaweb/models.py

```diff
@@ -978,6 +979,9 @@ class Network(models.Model):
     # Network environments let you specify things like "prod", "test", "dev",
     # etc so they can be filtered out of alerts. For internal use.
     environment = models.TextField(default="default")
+    # Added for Network Balance Limit
+    max_account_limit = models.BigIntegerField(default=10000)
```

Comment:
Does these fields need to be in the `PendingCreditUpdate` or can we query it from the associated `Network` ?